_data/collection-index.yml

```
@@ -822,4 +822,9 @@
   maintainer: Carsten Behring
   contact: https://github.com/scicloj/devcontainer-templates/issues
   repository: https://github.com/scicloj/devcontainer-templates/
-  ociReference: ghcr.io/scicloj/devcontainer-templates  
+  ociReference: ghcr.io/scicloj/devcontainer-templates
+- name: Synology devcontainer templates
```

```suggestion
- name: Synology devcontainer Templates
```

Matching with Dev Containers terminology.